arrow/arrow.py

```diff
@@ -295,6 +295,35 @@ def strptime(cls, date_str, fmt, tzinfo=None):
             fold=getattr(dt, "fold", 0),
         )
 
+    @classmethod
+    def fromordinal(cls, ordinal):
+        """Constructs an :class:`Arrow <arrow.arrow.Arrow>` object corresponding
+            to the Gregorian Ordinal.
+
+        :param ordinal: an ``int`` or a ``str`` that converts to an ``int``.
+
+        Usage::
+
+            >>> arrow.fromordinal(737741)
+            <Arrow [2020-11-12T00:00:00+00:00]>
+
+        """
+
+        if not util.is_ordinal(ordinal):
+            raise ValueError(f"The provided ordinal '{ordinal}' is invalid.")
+        dt = datetime.fromordinal(int(ordinal))
```

Comment:
Allowing an `str` argument might make this more complicated.

Reply:
I see, yes we could restrict this solely to integers. I was under the impression that string arguments might provide more usability? But maybe there are certain complications that aren't worth dealing with here?

Reply:
I think we should restrict it solely to integers and keep things simple for now.

Reply:
I agree. I will change it to restrict it to integers only.